apps/desktop-wallet/package.json

```diff
@@ -32,12 +31,14 @@
     "electron-is-dev": "^2.0.0",
     "electron-updater": "^5.3.0"
   },
+  "trustedDependencies": [
+    "electron",
+    "esbuild"
+  ],
```

Comment:
This was needed so that electon and esbuild can run their `postinstall` scripts. By default, bun does not run then. See more here: https://bun.sh/guides/install/trusted